ResearchKit/Common/ORKPasscodeStepViewController.m

```diff
@@ -39,9 +39,14 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import <AudioToolbox/AudioToolbox.h>
 #import <LocalAuthentication/LocalAuthentication.h>
 
+static CGFloat const kForgotPasscodeVerticalPadding     = 50.0f;
```

Comment:
Yes, I had the same concern. It's a very complicated view hierarchy to tap into and it does display properly on iPhone 4s up to iPhone 6s Plus (in simulator, anyway).